packages/admin/admin-icons/src/CometColor.tsx

```diff
@@ -1,23 +1,31 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
 
+function getUniqueId() {
+    return Math.random().toString(36).substring(7);
+}
+
 export default function Comet(props: SvgIconProps) {
+    const uniqueIdA = getUniqueId();
+    const uniqueIdB = getUniqueId();
+    const uniqueIdC = getUniqueId();
+
     return (
         <SvgIcon {...props} viewBox="0 0 200 200">
             <defs>
                 <path
-                    id="comet-icon-a"
```

Comment:
I'd keep the comet-icon-a prefix and add an UUID, like they're doing here: https://css-tricks.com/youre-inlining-svg-icons-deal-unique-titles-ids/